src/controllers/API.ts

```diff
@@ -194,6 +194,10 @@ export default class APIController {
       ...request,
       queryParams: {
         ...parseQueryParams(request.queryParams),
+        include: getQueryParamValue('include',request.rawQueryString)
```

Comment:
I get why you pulled this out here -- `parseQueryParams` is given only the `queryParams`, and we need the `rawQueryString` -- but I'd rather keep it where the only parameters that are parsed in the API controller are those that support custom parsing logic (which should eventually include `page`), whereas all those that have their parsing rules defined by json:api are pared in `parseQueryParams`. Can we instead just pass `request.rawQueryString` into `parseQueryParams` and go back to parsing `include` there?

Reply:
Ive moved it back into `parseQueryParams`, however its now appended at the end instead of being included in the generation of `paramsToParserFns`. I did it this way, so that the `paramsToParserFns` and parsing of `page` and `fields` stays the same, whereas `include` is the special case currently for standard query params. Is this approach suitable?

Reply:
Yup, this works for me. Thanks!